helpers/node-mark-offline

```diff
@@ -73,9 +74,38 @@ elif [[ "$NHC_RM" == "slurm" ]]; then
                         exit 0
                     fi
                     ;;
+                boot*)
+                    # Offline node after reboot if vanilla `scontrol reboot` was
+                    # called, so jobs can't run until NHC onlines the node.
+                    # Note: This won't happen while node is waiting to boot,
+                    # because $STATUS would show MIX@ or ALLOC@, not BOOT.
+                    # See src/common/slurm_protocol_defs.c-->node_state_string()
+                    SHOW_NODE_OUTPUT="$($SLURM_SCONTROL show node $HOSTNAME)"
+                    if [[ $SHOW_NODE_OUTPUT == *"State=REBOOT "* ]]; then
```

Comment:
I had to change this to `"State=REBOOT"` without the trailing space.  While a node was booting this is the state I saw: `State=REBOOT*+DRAIN`.